src/main/java/org/primefaces/component/inputtext/InputTextRenderer.java

```diff
@@ -73,6 +75,7 @@ protected void encodeMarkup(FacesContext context, InputText inputText) throws IO
             writer.writeAttribute("value", valueToRender, null);
         }
 
+        ComponentUtils.encodeDirection(context, inputText);
```

Comment:
@Rapster I see what you did here and you comment but I would still vote for renderRtlDirection() method to be consistent with others.  I know that putting that in the base class feels weird but not if we have more and more components using it?  I will wait for @tandraschko thoughts.

Reply:
@melloware yes I will reuse same naming convention